src/newsDbService.js

```diff
@@ -1,35 +1,48 @@
 const { Article, Source } = require('./database/mongooseSchema')
 
 module.exports = {
-	saveArticles: async articles => {
-		try {
-			return Article.insertMany(articles, { ordered: false })
-		} catch (error) {
-			if (error.code === 11000 || error.code === 11001) {
-				console.log('ignored duplicates')
-			} else {
-				console.log(error)
-			}
-		}
-		return null
-	},
+  saveArticles: async articles => {
+    try {
+      return Article.insertMany(articles, { ordered: false })
+    } catch (error) {
+      if (error.code === 11000 || error.code === 11001) {
+        console.log('ignored duplicates')
+      } else {
+        console.log(error)
+      }
+    }
+    return null
+  },
 
-	deleteArticles: async conditions => {
-		return Article.deleteMany(conditions)
-	},
+  saveArticle: async article => {
```

Comment:
@shrijan-siris is there a reason to create `saveArticle ` function instead of using `saveArticles` ?

Reply:
Explained in standup